tests/test_main.py

```
@@ -1411,6 +1577,467 @@ async def mock_sync_forever(*args, **kwargs):
+    """Unknown backend selection should clear the active backend."""
+
+    class _StubRegistry:
+        def __init__(self) -> None:
+            """
+            Initialize the event stub.
+            
+            Creates an empty list `set_calls` used to record values passed to `set()` (or `None` when no value is provided).
+            """
+            self.set_calls: list[str | None] = []
+
+        def register_backend(self, _backend: Any, *, replace: bool = False) -> None:
```

Ruff (0.14.14)

Unused method argument: `replace`

(ARG002)